prime-router/build.gradle.kts

```diff
@@ -162,9 +163,14 @@ tasks.test {
     }
 }
 
+tasks.dokkaHtml.configure {
```

Comment:
Suggestion: Replace the existing javadoc task with this or make it throw an error/message, so people are not confused.